pkg/build/gobuild.go

```diff
@@ -56,18 +58,24 @@ const (
 
 	defaultGoBin = "go"         // defaults to first go binary found in PATH
 	goBinPathEnv = "KO_GO_PATH" // env lookup for optional relative or full go binary path
+
+	envTemplateKey       = "Env"
+	gitTemplateKey       = "Git"
+	dateTemplateKey      = "Date"
+	timestampTemplateKey = "Timestamp"
```

Comment:
Probably just personal style, but if these consts are only used once as map keys below, I think it would be easier to read by just inlining them down there. Otherwise when reading the code below you have to scroll back up to here to tell what `envTemplateKey` actually is.